src/zjs_sensor.c

```diff
@@ -49,6 +49,7 @@ typedef struct sensor_handle {
 
 static sensor_handle_t *accel_handles = NULL;
 static sensor_handle_t *gyro_handles = NULL;
+static sensor_handle_t *light_handles = NULL;
```

Comment:
Is zjs_sensor where the light sensor stuff should live?  I thought that file contained sensors found built into the arduino 101, not sure we should add grove ones to it.